tools/integration_tests/concurrent_operations/setup_test.go

```diff
@@ -63,6 +63,10 @@ func TestMain(m *testing.M) {
 	flagsSet := [][]string{
 		{"--kernel-list-cache-ttl-secs=-1"}, {"--kernel-list-cache-ttl-secs=0"},
 	}
+	if !testing.Short() {
```

Comment:
Can we use this instead? - https://github.com/GoogleCloudPlatform/gcsfuse/blob/3dd81acc0034fc94ad047467a2af76d52effc927/tools/integration_tests/util/setup/setup.go#L495

Reply:
Done.